plugins/scaffolder-backend-module-rails/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@backstage/plugin-scaffolder-backend-module-rails",
-  "version": "0.5.1-next.2",
```

Comment:
Don't need these version changes here, you might just need to rebase :pray:

Reply:
oops, my bad! :)